va/caa.go

```diff
@@ -116,7 +117,61 @@ func parseResults(results []caaResult) (*CAASet, error) {
 	return nil, nil
 }
 
-func (va *ValidationAuthorityImpl) parallelCAALookup(ctx context.Context, name string, lookuper func(context.Context, string) ([]*dns.CAA, error)) []caaResult {
+// If the fqdn has a direct alias among the list of CNAMEs, return it; otherwise
+// return "". Note that RFC 6844 only asks for tree-climbing on direct aliases,
+// not recursive aliases.
+func findAlias(source string, cnames []*dns.CNAME) string {
+	for _, cname := range cnames {
+		if cname.Header().Name == source {
+			return cname.Target
+		}
+	}
+	return ""
+}
+
+func parent(fqdn string) string {
```

Comment:
`parent` could use a short comment to describe its parameter & return.

Reply:
Wound up replacing this with `parentDomains`, which has a comment.